src/range.js

```diff
@@ -24,9 +28,10 @@ module.exports = _curry2(function range(from, to) {
   }
   var result = [];
   var n = from;
-  while (n < to) {
+  var dx = from < to ? 1 : -1;
+  while (n !== to) {
```

Comment:
As discussed on Gitter, this would fail if `to` is not an integer (or more precisely if `to - from` when cast to a number is not an integer... gotta love JS.)